src/app/pages/HomePage/index.tsx

```diff
@@ -156,15 +161,14 @@ export const HomePage: FC = () => {
           </ThemeByNetwork>
         </Content>
 
-        <FooterStyled>
-          {showInfoScreenBtn && (
-            <IconButton aria-label={infoAriaLabel} onClick={onToggleInfoScreenClick}>
-              <InfoOutlinedIcon fontSize="medium" sx={{ color: 'white' }} />
-            </IconButton>
-          )}
-          {!isMobile && <Footer />}
-        </FooterStyled>
+        <FooterStyled>{!isMobile && <Footer />}</FooterStyled>
```

Comment:
I would simplify to `{!isTablet && <FooterStyled><Footer /></FooterStyled>` and remove breakpoints from FooterStyled